messaging/src/main/java/org/axonframework/messaging/OnNextMessageStream.java

```diff
@@ -57,4 +56,20 @@ public Optional<Entry<M>> next() {
         next.ifPresent(onNext);
         return next;
     }
+
+    static class Single<M extends Message<?>> extends OnNextMessageStream<M> implements MessageStream.Single<M> {
+
+        /**
+         * Construct an {@link MessageStream stream} that invokes the given {@code onNext} {@link Consumer} each time a
+         * new {@link Entry entry} is consumed by the given {@code delegate}.
+         *
+         * @param delegate The delegate {@link MessageStream stream} from which each consumed {@link Entry entry} is
```

Comment:
This doc part states that it's about multiple entries, which isn't the case here. 
As with the other JavaDoc stuff, I'll adjust it accordingly.